src/app/shared/table/actions/table-create-action.ts

```diff
@@ -22,12 +22,13 @@ export class TableCreateAction implements TableAction {
     return this.action;
   }
 
-  protected create(component: ComponentType<unknown>, dialog: MatDialog, refresh?: () => Observable<void>) {
+  protected create(component: ComponentType<unknown>, dialog: MatDialog, refresh?: () => Observable<void>, userId?: string) {
```

Comment:
@LucasBrazi06 @ClaudeROSSI is this ok to add the userId as an optional parameter ?

Reply:
Okay but just make it more generic like 'dialogData: any' so you can pass data to the dialog and only the caller and the receiver will know what's inside.